### DISCUSSION TITLE
Integration of SSOs in OpenSlides

### DISCUSSION THREAD

**Discussion by linux-lukas** [OpenSlides/OpenSlides#5868] [Ideas]
In my view, it could make sense for OpenSlides to be able to offer SSO integration in the future, so that companies and associations can use it if necessary.

**Comment by tsiegleauq** [comment]
We already support SAML

**Reply by DerUtz** [reply]
where can I find a documentation for configuration?

**Reply by jsangmeister** [reply]
We will add it shortly. Progress will be tracked in https://github.com/OpenSlides/OpenSlides/issues/6642
